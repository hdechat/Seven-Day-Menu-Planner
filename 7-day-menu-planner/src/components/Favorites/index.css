.favorites-text {
  color: #878984;
}

@media screen and (max-width: 350px) {
  .display-favorites {
    width: 590px;
  }

  .favorites-text {
    display: none;
  }
}
