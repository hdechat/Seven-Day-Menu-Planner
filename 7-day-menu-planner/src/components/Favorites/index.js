import React from 'react';
import RecipeCardsDisplay from '../RecipeCardsDisplay';
import './index.css';

const Favorites = ({ favorites }) => {
  return (
    <div className="display-cards">
      <RecipeCardsDisplay recipeCards={favorites} />
    </div>
  )
}

export default Favorites;
