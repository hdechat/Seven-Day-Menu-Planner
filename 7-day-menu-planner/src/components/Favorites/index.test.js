import React from 'react';
import Favorites from './index.js';
import { shallow } from 'enzyme';

describe('FAVORITES', () => {
  let wrapper;
  let mockFavorites;

  beforeEach(() => {
    mockFavorites = [{title: 'recipe'}, {title: 'recipe'}];

    wrapper = shallow(<Favorites favorites={mockFavorites} />);
  });

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});