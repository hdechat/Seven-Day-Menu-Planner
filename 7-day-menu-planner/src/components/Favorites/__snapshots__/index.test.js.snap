// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAVORITES matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Favorites
    favorites={
        Array [
            Object {
              "title": "recipe",
            },
            Object {
              "title": "recipe",
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <RecipeCardsDisplay
        recipeCards={
                Array [
                        Object {
                          "title": "recipe",
                        },
                        Object {
                          "title": "recipe",
                        },
                      ]
        }
/>,
      "className": "display-cards",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "recipeCards": Array [
          Object {
            "title": "recipe",
          },
          Object {
            "title": "recipe",
          },
        ],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <RecipeCardsDisplay
          recipeCards={
                    Array [
                              Object {
                                "title": "recipe",
                              },
                              Object {
                                "title": "recipe",
                              },
                            ]
          }
/>,
        "className": "display-cards",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "recipeCards": Array [
            Object {
              "title": "recipe",
            },
            Object {
              "title": "recipe",
            },
          ],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
